VideoLocker/res/values/styles.xml

```diff
@@ -106,6 +105,10 @@
         <item name="android:background">@color/edx_brand_primary_base</item>
     </style>
 
+    <style name="course_dashboard_separator_style" parent="separator_style">
```

Comment:
I think these different separator styles should be named according to color, to match the text styles.